src/index.js

```diff
@@ -1,14 +1,27 @@
-import _ from 'lodash';
 import './style.css';
 
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+const list = [
+  {
+    index: 0,
+    description: 'Do laundry',
+    completed: false,
+  },
+  {
+    index: 1,
+    description: 'Do laundry',
+    completed: false,
+  },
+  {
+    index: 2,
+    description: 'Do Challenge',
+    completed: false,
+  },
+
+];
```

Comment:
- [ ] `SUGGESTION` to fix the dragging position bug, you can make the `list` variable load from localStrorage _or_ use the hardcoded list, that way, if there's nothing in localStorage, it will load the hardcoded list

Example:

```js
const number = null || 55;
console.log(number) // outputs: 55
```